backend/tsconfig.json

```diff
@@ -2,7 +2,9 @@
    "extends": "@strapi/typescript-utils/tsconfigs/server",
    "compilerOptions": {
       "outDir": "dist",
-      "rootDir": "."
+      "rootDir": ".",
+      "jsx": "react",
+      "lib": ["ES2020", "DOM"]
```

Comment:
Why did we need to modify the `tsconfig` with this Strapi update?

Reply:
As weird as it sounds, when you're running certain Strapi commands, such as `strapi export`, they invoke a Typescript compilation, and that throws errors without these additional settings. 
I've also opened an issue on the Strapi repo to clarify this behavior https://github.com/strapi/strapi/issues/15707